### PR TITLE
Feature/ads

### PR DESCRIPTION
**Pull request by peterpf**
- Datasource can be switched to query ads data.
- Added ad attributes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,17 +26,17 @@ Configure the metric with following settings:
...
+| Filter*   | Apply filters to the query                                             |
 | Format as | `time_series`                                                          |
 | Group By* | Group query by dimension and plot as individual time series            |
 | Order By* | Order query ascending or descending by dimension                       |
 | Limit*    | Limit the result to given number                                       |
 | Interval  | Time granularity of queries, select `AUTO` to adapt to global interval |
 | Alias By* | Name of series (ignored when `Group By` is set)                        |
 
-Settings marked with * are optionally.
+Settings marked with `*` are optionally.
```
Comment (Tigraine) [inline README.md]:
```suggestion
Settings marked with `*` are optional.
```

File: src/datasource.js [hunk 2]
```diff
@@ -77,9 +78,12 @@ export class BitmovinAnalyticsDatasource {
       }
       data['groupBy'] = target.groupBy;
       data['limit'] = Number(target.limit) || undefined;
-
+      var apiRequestUrl = this.url + '/analytics/queries';
+      if (this.isAdAnalytics === true) {
```
Comment (Tigraine) [inline src/datasource.js]: move this to a method. doing two `var apiRequestUrl` is bad form

Comment (peterpf) [reply src/datasource.js]: Moved to method

File: src/datasource.js [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 import { AGGREGATION } from './types/aggregations';
-import {calculateAutoInterval, QUERY_INTERVAL} from './types/intervals';
+import { calculateAutoInterval, QUERY_INTERVAL } from './types/intervals';
 import { transform } from './result_transformer';
 import { ResultFormat } from './types/resultFormat';
 
+const getApiRequestUrl = (baseUrl, isAdAnalytics) => {
+  var apiRequestUrl = baseUrl + '/analytics/queries';
+  if (isAdAnalytics === true) {
+    apiRequestUrl = baseUrl + '/analytics/ads/queries';
```
Comment (Tigraine) [inline src/datasource.js]: can be simplified with a `return`

**Comment by peterpf** [comment]
https://github.com/bitmovin/analytics-issues/issues/657

**Pull request merged**